src/screens/Vault/AddReserveKey.tsx

```diff
@@ -32,11 +40,16 @@ import SuccessIcon from 'src/assets/images/successSvg.svg';
 import WalletUtilities from 'src/services/wallets/operations/utils';
 import { INHERITANCE_KEY1_IDENTIFIER } from 'src/services/wallets/operations/miniscript/default/InheritanceVault';
 
-const DEFAULT_INHERITANCE_TIMELOCK = { label: MONTHS_12, value: 12 * 30 * 24 * 60 * 60 * 1000 };
+const DEFAULT_INHERITANCE_TIMELOCK = { label: MONTHS_3, value: 3 * 30 * 24 * 60 * 60 * 1000 };
 const INHERITANCE_TIMELOCK_DURATIONS = [
   DEFAULT_INHERITANCE_TIMELOCK,
+
+  { label: MONTHS_6, value: 6 * 30 * 24 * 60 * 60 * 1000 },
```

Comment:
These values are both wrong and unused, updatw the value to just be 'MONTHS_12', 'MONTHS_18' etc.